docs/reference/gqlalchemy/transformations/translators/nx_translator.md

```diff
@@ -31,9 +31,9 @@ Generates Cypher queries for creating a graph by query groups.
 class NxTranslator(Translator)
 ```
 
-Uses original ids from Memgraph. Labels are encoded as properties. Since Networkx allows
+Uses original ids from Memgraph. Labels are encoded as properties. Since NetworkX allows
```

Comment:
There are additional instances of me changing the case of `NetworkX`, e.g. here...